src/System.IO.FileSystem/tests/DirectoryInfo/CreateSubdirectory.cs

```diff
@@ -211,6 +210,17 @@ public void UNCPathWithOnlySlashes()
             Assert.Throws<ArgumentException>(() => testDir.CreateSubdirectory("//"));
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void ParentDirectoryNameAsPrefix()
+        {
+            string randomName = GetTestFileName();
+            var di = new DirectoryInfo(Path.Combine(TestDirectory, randomName));
```

Comment:
simpler to write `Directory.CreateDirectory(Path.Combine(TestDirectory, randomName))`?